### PR TITLE
Set meta tags for fb and twtr

### PR DESCRIPTION
**Pull request by sheetaluk**
Set meta tags for twtr and fb.

When a user shares direct links on twitter and facebook, we want to
attach relevent information like Title, description,URL, Image etc
for the twitter and facebook scrapers to pick up and display in
tweets and shares.

### CODE REVIEW

File: bouncer/templates/base.html.jinja2 [hunk 1]
```diff
@@ -2,6 +2,19 @@
   <html lang=en>
   <head>
     <meta charset=utf-8>
+
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@hypothes_is" />
+    <meta name="twitter:title" content="{{ meta_title|safe }}" />
+    <meta name="twitter:description" content="Follow the link to see this annotation on the original page.  Use hypothes.is to annotate the Web and PDFs." />
```
Comment (robertknight) [inline bouncer/templates/base.html.jinja2]: When the organization's name is written in text, as opposed to a logo, I believe it should be written as "Hypothesis".

File: bouncer/templates/annotation.html.jinja2 [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {% extends "templates/base.html.jinja2" %}
 
 {% set title = _("Hypothesis: Annotation for %(site)s" )|format(site=pretty_url)  %}
+{% set meta_title = _("Annotation for %(site)s" )|format(site=pretty_url)  %}
+{% set meta_url = _("%(url)s" )|format(url=meta_url)  %}
```
Comment (seanh) [inline bouncer/templates/annotation.html.jinja2]: I don't see where this `meta_url` is being used?

**Comment by seanh** [comment]
Looks good, I just have two questions:

1. The `meta_url` variable doesn't seem to be used?
2. Do we want to copy the two image files into this repo so that they're hosted by bouncer?

**Comment by seanh** [comment]
Awesome! I don't want to deploy on a Friday evening, but I'll set a Slack reminder to do it on Monday morning. Or do you want to deploy this to staging then production yourself?

**Comment by sheetaluk** [comment]
YES I want to do it myself! but on monday :)

**Comment by nickstenning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;u6OaYcwt&#x2F;320-as-a-user-when-i-share-a-direct-link-on-twitter-fb-relevant-h-information-shows-up-on-my-news-feeds"> As a user, when I share a direct link on twitter&#x2F;fb, relevant H information shows up on my news feeds.</a>

**Pull request merged**